packages/core/src/errors/utxo.ts

```diff
@@ -0,0 +1,32 @@
+import { BaseError } from './base.js'
+
+export type UTXOsFetchErrorType = UTXOsFetchError & {
+  name: 'UTXOsFetchError'
+}
+
+export class UTXOsFetchError extends BaseError {
+  constructor({ address }: { address: string }) {
+    super(`There was a problem fetching UTXOs for address: ${address}.`, {
+      name: 'UTXOsFetchError',
+    })
+  }
+}
```

Comment:
And for this `UTXOsNotFoundError` maybe?

Reply:
But a NotFound error is different from a Fetch error.

NotFound is like a 404, and Fetch error is like a 500